src/app/layout/account-switcher.component.html

```diff
@@ -53,7 +53,6 @@
       <button
         *ngFor="let a of accounts | keyvalue"
         class="account"
-        [ngClass]="{ active: a.value.profile.authenticationStatus == 'active' }"
```

Comment:
This class will never be applied because the active account is excluded from the switcherAccount list. Also doesn't appear to have any scss styling associated with it.